tests/nightly/broken_link_checker_test/JenkinsfileForBLC

```diff
@@ -0,0 +1,74 @@
+// -*- mode: groovy -*-
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+
+//This is a Jenkinsfile for the broken link checker test.
+
+err = null
+
+def init_git() {
+  deleteDir()
+  retry(5) {
+    try {
+      timeout(time: 15, unit: 'MINUTES') {
+        checkout scm
+        sh 'git submodule update --init --recursive'
+        sh 'git clean -d -f'
+      }
+    } catch (exc) {
+      deleteDir()
+      error "Failed to fetch source codes with ${exc}"
+      sleep 2
+    }
+  }
+}
+
+
+try {
+  stage('BLC'){
+    parallel 'BrokenLinkChecker: CPU': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/brokenLinkChecker') {
+          init_git()
+          withCredentials([usernamePassword(credentialsId: 'github-leleamol', passwordVariable: 'APACHE_PASSWORD', usernameVariable: 'APACHE_USERNAME')]) {
+		  sh 'aws s3 cp s3://mxnet-ci-prod-slave-data/url_list.txt  ./tests/nightly/broken_link_checker_test/url_list.txt'
+          sh "ci/build.py --platform ubuntu_blc /work/runtime_functions.sh broken_link_checker"
```

Comment:
Please use the docker_run from the main Jenkinsfile. Otherwise, this run will not use our docker cache and slow down the run.